app/serializers/participant_serializer.rb

```diff
@@ -16,6 +16,17 @@ def active_participant_attribute(attr, &blk)
     def participant_active?(user)
       user.teacher_profile.ecf_profile&.active?
     end
+
+    def trn(user)
+      user.teacher_profile.trn || user.teacher_profile.ecf_profile.ecf_participant_validation_data&.trn
+    end
+
+    def validated_trn(user)
+      eligibility_status = user.teacher_profile.ecf_profile.ecf_participant_eligibility&.status
+      if %w[matched eligible].include?(eligibility_status)
+        user.teacher_profile.trn
+      end
+    end
```

Comment:
So, if there is an existing TRN from NPQ and we validate a different TRN, we always use the existing TRN not the validated one?

Reply:
Although in that case we would be `:manual_check` won't we?

Reply:
Yep - so we will share the NPQ one and `teacher_reference_number_validated`  will be false